CryptKeeperZK/crypt-keeper-extension
# fix: app styles and e2e speed up

## Explanation

This PR fixes minor style issues and speeds up e2e tests.

Details are below:
- [x] Rename Metamask to MetaMask to match the same naming
- [x] Fix styles for home lists
- [x] Add storage removal function for e2e builds
- [x] Speed up e2e tests run
- [x] Add backup onboarding test

## More Information

Blocked by #666 
Related to #330

## Screenshots/Screencaps

Related only to extended mode in e2e tests

## Manual Testing Steps

1. Check onboarding, login, upload styles
2. Check connect identity modal styles
3. Check home page styles

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
